ckanext/xloader/plugin.py

```diff
@@ -40,7 +40,7 @@ def is_it_an_xloader_format(cls, format_):
 class xloaderPlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IConfigurer)
     plugins.implements(plugins.IConfigurable)
-    plugins.implements(plugins.IResourceUrlChange)
```

Comment:
we could check for the validation plugin here and then we don't need the rest of the changes in the PR, right?

Reply:
or rather, we could keep this implements line and add the check for the plugin in the after_create function below